### DISCUSSION TITLE
Roadmaps?

### DISCUSSION THREAD

**Discussion by citrus327** [web-infra-dev/rspress#103] [General]
I'm currently developing a React component documentation tool, and it's pretty much like Rspress, but relying on Vite.

I'd like to see a roadmap for Rspress. The project itself seems like are for infra team only (or something like this).

People might don't have any clues about what to contribute, since most of the feature and issues are assigned.

 Showing us a roadmap might be good for somebody like me.

**Comment by chenjiahan** [comment]
Rspress does indeed need a public roadmap, as it can encourage more contributors to participate in the development of Rspress. @sanyuan0704 , can you provide one?

**Reply by sanyuan0704** [reply]
I will share the roadmap recently and thank you for feedback @citrus327

**Reply by chenjiahan** [reply]
We can refer to Rspack and create a quarterly Iteration plan: https://github.com/web-infra-dev/rspack/issues/3679

**Comment by sanyuan0704** [comment]
Here is the Rspress roadmap https://github.com/web-infra-dev/rspress/issues/133 .
